### PR TITLE
LPS-122886 now with uniqueFileName we can get the title without the extension

### PR DESCRIPTION
**Pull request by AliciaGarciaGarcia**

### CODE REVIEW

**Comment by AliciaGarciaGarcia** [comment]
ci:test:sf

**Comment by AliciaGarciaGarcia** [comment]
ci:test:relevant

File: portal-impl/src/com/liferay/portlet/documentlibrary/util/DLImpl.java [hunk 1]
```diff
@@ -864,22 +864,23 @@ public String getTitleWithExtension(String title, String extension) {
 	public String getUniqueFileName(
```
Comment (4lejandrito) [inline portal-impl/src/com/liferay/portlet/documentlibrary/util/DLImpl.java:864]: Should we do the same in  [DefaultUniqueFileNameProvider](https://github.com/liferay-lima/liferay-portal/blob/091bf6a11e607f516e7852e792d94d30387d5c55/modules/apps/upload/upload-web/src/main/java/com/liferay/upload/web/internal/DefaultUniqueFileNameProvider.java)?

Comment (AliciaGarciaGarcia) [reply portal-impl/src/com/liferay/portlet/documentlibrary/util/DLImpl.java:864]: Yes, I have a task on my notebook to review Google Drive and OneDrive behaviour and it is when this must be changed, on my opinion

Comment (AliciaGarciaGarcia) [reply portal-impl/src/com/liferay/portlet/documentlibrary/util/DLImpl.java:864]: Well, better create a story for that, Thanks for pointing at it!

**Review by adolfopa — APPROVED** [review]
Looks good to me 👍

**Comment by AliciaGarciaGarcia** [comment]
ci:forward

**Pull request closed without merging**